pkg/controller/zerocapacity/zero_controller.go

```diff
@@ -217,31 +216,23 @@ func (z *Controller) initializeResources(request reconcile.Request, instance Res
 	}
 
 	if err := ensureClusterStability(infinispan); err != nil {
-		return reconcile.Result{}, fmt.Errorf("Infinispan not stable: %w", err)
-	}
-
-	statefulset := &appsv1.StatefulSet{}
```

Comment:
Good spot. I forgot to remove this when I updated how the zero pod was configured.